example/20-componentconfig-gardenlet.yaml

```diff
@@ -52,6 +52,7 @@ controllers:
     staleExtensionHealthChecks:
       enabled: true
     # threshold: 5m
+    managedResourceProgressingThreshold: 1h
```

Comment:
Should we also add this for the seed-care controller? Can't we face similar problems there as well?

Reply:
There was no such case observed in past  that's why I didn't add it.

Reply:
OK, but conceptionally it can happen there as well, so let's better add it now since we touch this code anyways instead of having follow-up effort later.

Reply:
For seed, we already check MR progressing condition, https://github.com/gardener/gardener/blob/d9ac107ea424bae2200ecfa0b93a84bd4c956f0e/pkg/operation/care/checker.go#L255-L259
https://github.com/gardener/gardener/blob/d9ac107ea424bae2200ecfa0b93a84bd4c956f0e/pkg/operation/care/seed_health.go#L146-L154
 So if MR is in progressing for more than `SeedSystemComponentsHealthy` threshold MR is considered in failed state and subsequently, `SeedSystemComponentsHealthy` is considered to be in failed state.
 Should I keep this behavior or harmonize it to match it with shoot care controller and use `managedResourceProgressingThreshold` for marking progressing MR as failed? I prefer the second option.

Reply:
I just checked again what's happening, and this is what I understand:

For `Seed`s, if a `ManagedResource` is `Progressing` then the `SeedSystemComponentsHealthy` also becomes `Progressing`, and if this takes longer than `1m` we mark it as false, I think. We don't want to keep `SeedSystemComponentsHealthy` as `True` if there are `ManagedResource` with `healthy=progressing=true` since we need to ensure that the seed is healthy and up to date before we start a `Shoot` reconciliation. 

In this light, the change of this PR does not seem to make sense for the seed-care controller. WDYT?

Reply:
> For Seeds, if a ManagedResource is Progressing then the SeedSystemComponentsHealthy also becomes Progressing, and if this takes longer than 1m we mark it as false, I think. We don't want to keep SeedSystemComponentsHealthy as True if there are ManagedResource with healthy=progressing=true since we need to ensure that the seed is healthy and up to date before we start a Shoot reconciliation.

Yes, But I was suggesting harmonizing things every where and using `managedResourceProgressingThreshold ` field to mark MR unhealthy/failed, and based on it `SeedSystemComponentsHealthy` will be marked unhealthy/failed.

But this will increase overall complexity without adding any value. And it is better to not add this field in the seed care controller configuration.